src/main/java/nextstep/courses/domain/session/SessionType.java

```diff
@@ -0,0 +1,10 @@
+package nextstep.courses.domain.session;
+
+public abstract class SessionType {
+
+    public abstract boolean isFree();
+
+    public abstract boolean isOverCapacity(int currentEnrollmentCount);
+
+    public abstract boolean isValidPayment(Long paymentAmount);
```

Comment:
```suggestion
public interface class SessionType {

    boolean isFree();

    boolean isOverCapacity(int currentEnrollmentCount);

    boolean isValidPayment(Long paymentAmount);
```
추상 메서드만 가진다면 인터페이스로 구현 가능함

Reply:
다른 메서드가 있었는데 정리하다보니 추상메서드만 남았네요 👍